vite.config.js

```diff
@@ -12,12 +13,27 @@ export default defineConfig({
   plugins: [
     react(),
     eslintPlugin({
-      cache: false, // disable eslint cache to avoid conflicts
+      cache: false,
     }),
+    nodePolyfills(),
   ],
   build: {
     sourcemap: true,
     emptyOutDir: true,
+    rollupOptions: {
+      onwarn: (warning, defaultHandler) => {
+        // TODO: remove isFromKbssCvutPackageWarning and solve the root cause, see https://github.com/kbss-cvut/record-manager-ui/issues/113
+        const isFromKbssCvutPackageWarning =
+          warning.code === "EVAL" && warning.message.includes("node_modules/store/plugins/lib/json2.js");
+        const isRollupPureAnnotationWarning =
```

Comment:
```suggestion
        // TODO: Rollup Pure Annotation warning should be resolved by https://github.com/kbss-cvut/s-forms/issues/282 
        const isRollupPureAnnotationWarning =
```

Reply:
Left a comment [here](https://github.com/kbss-cvut/record-manager-ui/issues/113#issuecomment-1995671719), but we can continue in this thread

Reply:
I believe the comment above is not related to the line that I annotated here:
` const isRollupPureAnnotationWarning =`

I am committing my suggesting here.